test/e2e/sourcery.js

```diff
@@ -5,45 +5,90 @@ module.exports = {
         browser
         .url(browser.launchUrl)
         .waitForElementVisible('h1'); // wait for the Login title
-        //Click to expand select user dropdown
+        //Click to expand select user dropdown--Step 1
         browser.element('css selector', '#react-select-2--value', function(result) {
             if(result.status != -1) { 
                 browser.click('#react-select-2--value');
             }
         });
-        //Select from expanded droprown
+       // browser.pause(10000);
+        //Select from expanded dropdown--Step 2
         browser.element('css selector', '[aria-label="Demo User"]', function(result) {
             if(result.status != -1) { 
                 browser.click('css selector', '[aria-label="Demo User"]');
             }
         });
         //Assert value is selected
         browser.assert.containsText('#react-select-2--value-item', 'Demo User');
-        //Click to expand select role dropdown
+
+        //Click to expand select role dropdown--Step 3
         browser.element('css selector', '#react-select-3--value', function(result) {
             if(result.status != -1) { 
                 browser.click('css selector', '#react-select-3--value');
             }
         });
-        //Select from expanded droprown
+        //Select from expanded dropdown--Step 4
         browser.element('css selector', '[aria-label="Admin"]', function(result) {
             if(result.status != -1) { 
                 browser.click('css selector', '[aria-label="Admin"]');
             }
         });
         //Assert value is selected
         browser.assert.containsText('#react-select-3--value-item', 'Admin');
-        //Click submit button
+
+        //Click submit button--Step 5
         browser.element('css selector', '[type="submit"]', function(result) {
             if(result.status != -1) {
                 browser
                 .click('css selector', '[type="submit"]')
                 .waitForElementVisible('.user-info__title');
             }
         });
-        //Assert if expected user is logged in
-        browser.assert.containsText('.user-info__title', 'Demo User')
+
+        // First bullet point
+        browser.element('css selector', '[class="user-info__title"]', function(result) {
+            if(result.status != -1) {
+                browser.waitForElementVisible('.user-info__title');
+            }
+        });
+        browser.assert.containsText('.user-info__title','Demo User'); 
+
+        // Second bullet point
+        browser.element('css selector', 'a[href="/time-logging"]');
```

Comment:
What is the use of this line? Applies to all line where you get element but do nothing with it.

Reply:
What are the cases when after getting the element you do something with it? Its like clicks?